File: app/src/main/java/ble/ble/elibro/MainViewHolder.java
package ble.ble.elibro;

import android.view.View;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.cardview.widget.CardView;
import androidx.recyclerview.widget.RecyclerView;

public class MainViewHolder extends RecyclerView.ViewHolder{

    public TextView mTextView;
    public CardView mCardView;

    public MainViewHolder(@NonNull View itemView) {
        super(itemView);

        mTextView = itemView.findViewById(R.id.txtView);
        mCardView = itemView.findViewById(R.id.pdf_cardView);
    }
}

File: app/src/main/java/ble/ble/elibro/MainViewHolder.kt
package ble.ble.elibro

import android.view.View
import androidx.recyclerview.widget.RecyclerView
import android.widget.TextView
import androidx.cardview.widget.CardView
import ble.ble.elibro.R

class MainViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    var mTextView: TextView
    var mCardView: CardView

    init {
        mTextView = itemView.findViewById(R.id.txtView)
        mCardView = itemView.findViewById(R.id.pdf_cardView)
    }
}